liveblog/liveblog
# [LBSD-2809] - Error is thrown when trying to edit a syndicated poll

### Purpose

This PR fixes a bug which was causing updates on Polls to fail on the consumer side after syndication. This issue was caused because the consumer poll was being created on the items service instead of the poll service.  This PR also fixes a small bug of showing the ingestion indicator on the Producer side.

### What has changed

- Changed the `create_syndicated_blog_post` function to accommodate for the polls to be created by the polls service and not the items service before creating the post for the consumer. 
- Show ingestion indicator on Consumer side only.

### Steps to test

1. Log in to the Editor admin as a producer and create a poll.
2. Log in to the Editor admin as a consumer and ingest the new post from the producer.
3. Try editing the poll from the timeline i.e change the duration of the poll. Observe that it updates correctly. 


Resolves: [LBSD-2709](https://sofab.atlassian.net/browse/LBSD-2809)

[LBSD-2709]: https://sofab.atlassian.net/browse/LBSD-2709?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
